### DISCUSSION TITLE
[Support]: Home Assistant: "Repository for add-on Frigate NVR Proxy is missing"

### DISCUSSION THREAD

**Discussion by sco0ts** [blakeblackshear/frigate#15444] [General Support]
### Describe the problem you are having

My home assistant install is showing the following warning: Missing repository for an installed add-on
"Repository for add-on Frigate NVR Proxy is missing. This means it will not get updates, and backups may not be restored correctly as the supervisor may not be able to build/download the resources required.

Please check the [add-on's documentation](http://homeassistant.local:8123/hassio/addon/ccab4aaf_frigate-proxy) for installation instructions and add the repository to the store."

Can anyone advise how to fix this?

### Version

0.13.2-6476f8a

### What browser(s) are you using?

Chrome

### Frigate config file

```yaml
na
```

### Relevant Frigate log output

```shell
na
```

### Relevant go2rtc log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Install method

Docker Compose

### docker-compose file or Docker CLI command

```yaml
na
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

na

**Comment by NickM-27** [comment]
I'd suggest just following https://docs.frigate.video/frigate/installation#home-assistant-addon
